MetaMask/metamask-mobile
# Add Custom Network during on boarding

**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
Now it's possible to add a custom network during onboarding, that enables the user to choose not to use a infura rpc url.

**Screenshots/Recordings**
https://recordit.co/PIQjzWnaue

**Test Cases**
Case1:
* Fresh install the app
* Press settings on meta metrics screen
* add another rpc url for ethereum
* finish on boarding (create a new wallet and importing an existing one) 
* go to settings -> networks
* check active network and active network rpc url



**Issue**

Progresses #???

**Checklist**
